oemof_flexmex/plotting/draw.py

```diff
@@ -0,0 +1,173 @@
+
+
+import os
+import pdb
+
+import pandas as pd
+import matplotlib.pyplot as plt
+import numpy as np
+from collections import OrderedDict
+
+dir_name = os.path.abspath(os.path.dirname(__file__))
+colors_csv = pd.read_csv(
+    os.path.join(dir_name, "colors.csv"), header=[0], index_col=[0])
+
+colors_csv = colors_csv.T
+colors_odict = OrderedDict()
+for i in colors_csv.columns:
+    colors_odict[i] = colors_csv.loc["Color", i]
+
+
+# from analysis import colors
+
+"""
+Import a specific range of data from energy_statistical_countrydatasheets.xlsx
+"""
+def import_countrydatasheet_data(sheet_name, last_row_to_skip, number_of_rows):
+    data = os.path.join(os.path.dirname(__file__), '../data/energy_statistical_countrydatasheets.xlsx')
+    df = pd.read_excel(data, sheet_name=sheet_name, index_col=2, usecols="A:AG",
+                       skiprows=lambda x: x in range(0, last_row_to_skip) and x != 7, nrows=number_of_rows, engine='openpyxl')
+    df1 = df.drop([8, 'Unnamed: 1'], axis=1)
+    return df1, sheet_name
+
+"""
+Functions for creating stacked bar plots from a table in the scalars-table-format. The input data
+must contain only one single country.
+"""
+# TODO: use country as an input variable
+def preprocessing_stacked_scalars(plot_data, factor, onxaxes): # put a factor here that the values should be devided be, e.g. 1 or 1000
+    df_plot_conversion_heat = pd.DataFrame()
+    df_plot_conversion_electricity = pd.DataFrame()
+    df_plot_storage_heat = pd.DataFrame()
+    df_plot_storage_electricity = pd.DataFrame()
+    df_plot_capacity_heat = pd.DataFrame()
+    df_plot_capacity_electricity = pd.DataFrame()
+
+    if plot_data['Parameter'].str.contains('EnergyConversion_Capacity_Electricity').any():
```

Comment:
We decided to replace this nested if, else part by a more simple approach: Just pass the list of parameters you want to select.